package.json

```diff
@@ -283,6 +283,7 @@
     "yargs": "^17.7.2"
   },
   "resolutions": {
+    "elliptic": "^6.6.1",
```

Comment:
I had to use a resolution, because the funding scripts uses ethers 5 because of the mento-sdk. And ethers 5 has a fixed dependency on `elliptic@6.5.4`